src/main/scala/net/cakesolutions/CakeBuildPlugin.scala

```diff
@@ -85,13 +83,15 @@ object CakeBuildPlugin extends AutoPlugin {
       scalaOrganization.value % "scala-reflect" % scalaVersion.value,
       scalaOrganization.value % "scalap" % scalaVersion.value
     ),
+    packageOptions in (Compile, packageBin) +=
+      Package.ManifestAttributes(CakeBuildInfoKeys.generalInfo.toSeq: _*),
```

Comment:
is there a reason to duplicate this information across the manifest and the buildinfo object?